README.md

```diff
@@ -97,6 +97,10 @@ and `worker` as found under `master`.
 
 The utility will launch an interactive wizard to guide you through creating your environment.
 
+#### *isolated staging environments*
+
+The option to create isolated staging environments is provided during the interactive wizard. Isolated staging environment is done by creating new experiments using data from existing experiments. Data and record is copied independently from the creation of staging environments. Therefore, in the event of failure, you'll still have to [manually unstage copied resources](#removing-staging-resources). 
```

Comment:
It will be nice to add a sentence about why it can be useful for people to use this feature

Reply:
Added in commit 89ab6b0